index.html

```diff
@@ -645,6 +645,17 @@ <h2 id="demonstrations">Demonstrations</h2>
 }</pre>
 			</div>
 
+			<div class="demo">
+				Custome arrow symbol: <input id="date-range51" size="30" value="">
+				<a href="#" class="show-option">Show Config</a>
+				<pre class="options">
+{		
+    customArrowPrevSymbol: '&lt;i class="fa fa-arrow-circle-left"&gt;&#8592;&lt;/i&gt;',

```

Comment:
To make this demo work completely, the FontAwesome stylesheets are required. Please mention this in a comment.